src/test/java/stest/tron/wallet/onlineStress/TestStorageAndCpu.java

```diff
@@ -45,109 +64,118 @@ public void beforeSuite() {
-      /*      logger.info("fee " + infoById.get().getFee());
-      logger.info("CpuFee " + infoById.get().getReceipt().getCpuFee());
-      logger.info("cpuUsage " + infoById.get().getReceipt().getCpuUsage());
-      logger.info("Storagefee " + infoById.get().getReceipt().getStorageFee());
-      logger.info("storageDelta " + infoById.get().getReceipt().getStorageDelta());*/
-      String feeString = Long.toString(infoById.get().getFee());
-      String cpuFeeString = Long.toString(infoById.get().getReceipt().getEnergyFee());
-      String cpuUsageString = Long.toString(infoById.get().getReceipt().getEnergyUsage());
-      //String storageFeeString = Long.toString(infoById.get().getReceipt().getStorageFee());
-      //String storageDeltaString  = Long.toString(infoById.get().getReceipt().getStorageDelta());
-      String netUsageString = Long.toString(infoById.get().getReceipt().getNetUsage());
-      String netFeeString = Long.toString(infoById.get().getReceipt().getNetFee());
-      String energyTotal = Long.toString(infoById.get().getReceipt().getEnergyUsageTotal());
-      writeCsv(feeString,cpuFeeString,cpuUsageString, netUsageString,netFeeString,txid,energyTotal);
-
-      //PublicMethed.buyStorage(10000000000L,testAccount2Address,testAccount2Key,blockingStubFull);
-
-
-      //SmartContract smartContract = PublicMethed.getContract(contractAddress,blockingStubFull);
-
-      /*      accountResource1 = PublicMethed.getAccountResource(testAccount1Address,
-      blockingStubFull);
-      Long afterCpuUsage = accountResource1.getCpuUsed();
-      Long afterStorageUsed = accountResource1.getStorageUsed();
-      account = PublicMethed.queryAccount(testAccount1Key,blockingStubFull);
-      Long afterBalance = account.getBalance();
-      writeCsv(accountString,beforeCpuUsage,beforeStorageUsage,beforeBalance,afterCpuUsage,
-          afterStorageUsed,afterBalance);*/
-
-
-      i++;
     }
   }
 
+
   @AfterClass
   public void shutdown() throws InterruptedException {
-    if (channelFull != null) {
-      channelFull.shutdown().awaitTermination(5, TimeUnit.SECONDS);
-    }
-  }
-
-  public static void writeCsv(String feeString,String cpuFeeString,String cpuUsageString,
-      String netUsageString,
-      String netFeeString,String txid,String energyTotalString) {
+    afterTime = System.currentTimeMillis();
     try {
-      File csv = new File("/Users/wangzihe/Documents/test.csv");
-      String time = Long.toString(System.currentTimeMillis());
-      BufferedWriter bw = new BufferedWriter(new FileWriter(csv, true));
-      /*      bw.write(time + "," + feeString + "," + cpuFeeString + "," + cpuUsageString + ","
-          + netUsageString + ","
-          + netFeeString + "," + txid);*/
-      bw.write(time + "," + energyTotalString + "," + txid);
-      bw.newLine();
-      bw.close();
-    } catch (FileNotFoundException e) {
-      e.printStackTrace();
-    } catch (IOException e) {
+      Thread.sleep(10000);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this use of "Thread.sleep()". ([squid:S2925](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2925))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1678&issues=AWbiJDiULpqqSUlcrGtG&open=AWbiJDiULpqqSUlcrGtG)

[ISSUE_KEY]: AWbiJDiULpqqSUlcrGtG